heroku/cli
# refactor(redis-v5): Move 'redis:timeout' to oclif

### Description
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001axcLzYAI/view

This converts `redis:timeout` to oclif and updates its associated tests.

### Testing

1) Run `./bin/run redis:timeout -a APP -s SECONDS` with this branch pulled down with an app that has redis on it and seconds is something greater than 0
2) Verify the command works
3) Compare the results with prod cli command and make sure things look 👍
4) Repeat 1-3 with SECONDS is set to 0
